spec/result_merger_spec.rb

```diff
@@ -73,14 +72,23 @@
               expect(SimpleCov::ResultMerger.results.length).to eq(1)
             end
           end
+        end
+      end
 
-          context "with merging disabled" do
-            before { SimpleCov.use_merging false }
+      describe ".store_result" do
+        it "refreshes the resultset" do
+          set = SimpleCov::ResultMerger.resultset
+          SimpleCov::ResultMerger.store_result({})
+          new_set = SimpleCov::ResultMerger.resultset
+          expect(new_set).not_to equal(set)
+        end
+      end
 
-            it "returns nil for SimpleCov.result" do
-              expect(SimpleCov.result).to be_nil
-            end
-          end
+      describe ".resultset" do
+        it "caches by default" do
+          set = SimpleCov::ResultMerger.resultset
+          new_set = SimpleCov::ResultMerger.resultset
+          expect(new_set).to equal(set)
```

Comment:
Personally I prefer `be` over `equal` for that because `equal` is so close to `eq` and `eql` that it's way too confusing that one checks equality, the other identity. I think `be` communicates that better, I can also change it myself though :)